sycl/source/handler.cpp

```diff
@@ -2055,6 +2055,8 @@ void handler::SetHostTask(std::function<void(interop_handle)> &&Func) {
   setType(detail::CGType::CodeplayHostTask);
 }
 
+// TODO: This function is not used anymore, remove it in the next
+// ABI-breaking window.
 void handler::addAccessorReq(detail::AccessorImplPtr Accessor) {
```

Comment:
We should put it under the -fpreview-breaking-flag. Something like:
```
#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
void handler::addAccessorReq() {
...
}
#endif
```
We use this macro to track all ABI-breaking changes. So that, during the next ABI-breaking window, we can make the code under `ifdef __INTEL_PREVIEW_BREAKING_CHANGES` default.

Reply:
Done.